app/services/proofing/result.rb

```diff
@@ -65,5 +65,5 @@ def to_h
         success: success?,
       }
     end
-  end
-end
+    end
+      end
```

Comment:
this indentation seems wrong. I think it was correct before
```suggestion
  end
end
```